thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -27,11 +27,57 @@ export default Ember.Component.extend({
    */
   isLoading: false,
 
+  /**
+   * Columns for metrics table
+   * @type Object[]
+   */
+  metricsTableColumns: [
+    {
+      template: 'custom/table-checkbox'
+    }, {
+      propertyName: 'metric',
+      title: 'Metric Name',
+      className: 'rootcause-metric__table__column'
+    }, {
+      propertyName: 'score',
+      title: 'Anomalous Score',
+      className: 'rootcause-metric__table__column'
+    }, {
+      template: 'custom/metrics-table-changes',
+      propertyName: 'change',
+      title: 'Changes',
+      className: 'rootcause-metric__table__column'
+    }
+  ],
+
   init() {
     this._super(...arguments);
     this.setProperties({ sortProperty: ROOTCAUSE_METRICS_SORT_PROPERTY_CHANGE, sortMode: ROOTCAUSE_METRICS_SORT_MODE_ASC });
   },
 
+  /**
+   * Data for metrics table
+   * @type Object[] - array of objects, each corresponding to a row in the table
+   */
+  metricsTableData: Ember.computed(
+    'urns',
+    'metrics',
+    'scores',
+    'changes',
+    function() {
+      let arr = [];
+      const { urns, metrics, scores, changesFormatted } = this.getProperties('urns', 'metrics', 'scores', 'changesFormatted');
```

Comment:
You have a listener for 'changes' above but you grab a 'changesFormatted' here. Is that on purpose?

Reply:
Nice catch. I meant to listen for 'changesFormatted'. Will make the change.